### PR TITLE
XMIR guide

### PR DESCRIPTION
**Pull request by yegor256**

### CODE REVIEW

**Comment by yegor256** [comment]
@Graur please, take a look

**Review by Graur — APPROVED** [review]

File: _posts/2022/11/2022-11-25-xmir-guide.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  <errors>
+    <error severity="warning" line="3">There is an extra bracket</error>
+    <error severity="error" line="12">The object 'x' is not found</error>
+  </errors>
+</program>
+```
+
+The errors with the `warning` severity may more or less safely be ignored. The
+  errors with the `error` severity will lead to failures in further compilation
+  and processing.
```
[the remark this answers is not in the document]
Comment (yegor256) [reply _posts/2022/11/2022-11-25-xmir-guide.md]: @Graur right, added in 7973e10

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**